src/main.rs

```diff
@@ -1,9 +1,13 @@
 #![no_std]
 #![no_main]
 
+mod arch;
+
+#[allow(unused_imports)]
```

Comment:
Is this used because we don't have a panic handler yet?

Reply:
Yes